### ISSUE TITLE
About libconsole.so

### ISSUE THREAD

**Issue by LoserJL** [open-power/hostboot#235]
Hi, experts
Why is the console module compiled as BASE_ MODULES, but it is EXT_IMAGE in the g_exttaskinfolist?
<img width="1274" alt="3353f9fd56f262968922314e966626c3" src="https://github.com/open-power/hostboot/assets/33047076/7702de30-b55a-4daa-a053-89bd9e4bc2cd">

Thanks!

**Comment by LoserJL** [comment]
> Looks like a very old code bug.

I got it, thank you!

In VfsRp:_loadUnload function, if a specific module cannot be found in the extend module, it will be redetermined whether it is in the base module, so there will be no problem with execution

**Comment by dcrowell77** [comment]
Yeah, the code functions correctly as written, but things look funny and are confusing.  We're working on a cleanup.
